game_event/models.py

```diff
@@ -17,3 +20,45 @@ class GameEvent(models.Model):
         validators=[MinLengthValidator(4)],
         choices=BallGame.choices
     )
+
+    @staticmethod
+    def create_game_event_from_form(request):
```

Comment:
My intention was to create a function (similar to the one in #47) that incorporates validators.
This way, from the views, you can simply call the create method(which shouldnt require a request input, but only relevant fields for `GameEvent`). If an error occurs during the creation process, it can be caught in the views and a relevant error message can be sent

This way the creation of `GameEvent` will be more robust, and not force it to be in a request form

Reply:
i fixed it , but in this current approch only one messeage will be displayed to the user.. the first ValidationError.

Reply:
I suggest modifying the function to throw an exception with a string instead of a list. This will make it more user-friendly for future users who may expect a string output.
To achieve this, you can init errors to an empty string and concatenating relevant errors using `errors += 'relevant error'` instead of repeatedly appending to the list. 
Alternatively, if you want to keep the current Implementation you can use the join method to concatenate the errors list  
  `" ".join(errors)`.

What do you think?

Reply:
could you elaborate ? I didnt understand the difference in your approch.

Reply:
It's a little uncomfortable this way, cause i want to later iterate over the errors and print messeage to the user